drivers/soundwire/bus.c

```diff
@@ -1493,6 +1491,10 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 			clear |= SDW_SCP_INT1_IMPL_DEF;
 		}
 
+		/* the SDCA interrupts are cleared in the codec driver .interrupt_callback() */
+		if (slave->prop.is_sdca && sdca_cascade)
+			slave_notify = true;
```

Comment:
nitpick: if `slave->prop.is_sdca` is false, `scda_cascade` will never be set, so it'll remain with its initial value of `false`. So it's enough to just test `if (sdca_cascade)` here

Reply:
fixed